shell/platform/darwin/ios/ios_context_metal.h

```diff
@@ -24,7 +24,7 @@ class IOSContextMetal final : public IOSContext {
 
   fml::scoped_nsobject<FlutterDarwinContextMetal> GetDarwinContext() const;
 
-  sk_sp<GrDirectContext> GetMainContext() const;
```

Comment:
So the idea is that there's 2, a main rendering context on the raster thread and a resource secondary context on the io thread? If so, be fully descriptive in the doc so someone new can understand.

Reply:
updated docstring in ios_context, added link to it.